test/powershell/engine/Basic/SemanticVersion.Tests.ps1

```diff
@@ -22,6 +22,14 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             $v.BuildLabel | Should -BeNullOrEmpty
             $v.ToString() | Should -Be "1.0.0"
 
+            $v = [SemanticVersion]::new("1.2.3+BLD.a1-xxx.03")
+            $v.Major | Should -Be 1
+            $v.Minor | Should -Be 2
+            $v.Patch | Should -Be 3
+            $v.PreReleaseLabel | Should -BeNullOrEmpty
+            $v.BuildLabel | Should -Be "BLD.a1-xxx.03"
+            $v.ToString() | Should -Be "1.2.3+BLD.a1-xxx.03"
+
```

Comment:
Can you add a test case to cover `1.0.0+META`? That's what you fixed in this PR but there is no test for it.

Reply:
Oh sure. Anything after the '+' is considered metadata (or META) which is why I thought this test case was appropriate.

Reply:
But now that I look back over the code. Yes, I wasn't testing the change. Thanks!